tests/components/yale_smart_alarm/test_config_flow.py

```diff
@@ -0,0 +1,135 @@
+"""Test the Yale Smart Living config flow."""
+from __future__ import annotations
+
+from unittest.mock import patch
+
+from yalesmartalarmclient.client import AuthenticationError
+
+from homeassistant import config_entries, setup
+from homeassistant.components.yale_smart_alarm.const import DOMAIN
+from homeassistant.core import HomeAssistant
+
+
+async def test_form(hass: HomeAssistant) -> None:
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.yale_smart_alarm.config_flow.YaleSmartAlarmClient",
+        return_value=True,
```

Comment:
This shouldn't return True. We're patching the client class.

Reply:
Yepp, have removed it from all three places